### PR TITLE
Add data type info to the Erlang cheatsheet

### PR DESCRIPTION
**Pull request by devonestes**
Hopefully this makes getting up and running a bit faster for folks
trying out Gleam who have some Erlang experience.

### CODE REVIEW

File: src/cheatsheets/gleam-for-erlang-users.md [hunk 1]
```diff
@@ -192,3 +192,163 @@ fn main() {
...
+#### Gleam
+
+```rust,noplaypen
+"Hellø, world!"
+```
+
+### Tuples
+
+Unlike in Erlang, there is no tuple literal syntax in Gleam, but they are very useful in Gleam as
+they're the only collection data type that allows for mixed types of elements in the collection.
```
Comment (lpil) [inline src/cheatsheets/gleam-for-erlang-users.md:216]: This isn't quite right, we have a tuple literal syntax in Gleam! You've used it below :)

File: src/cheatsheets/gleam-for-erlang-users.md [hunk 2]
```diff
@@ -192,3 +192,163 @@ fn main() {
...
+#### Gleam
+
+```rust,noplaypen
+type Person {
+  Person(age: Int, name: String)
+}
+
+let person = Person(name: "name", age: 35)
+
+let Person(name: name, age: _) = person
```
Comment (lpil) [inline src/cheatsheets/gleam-for-erlang-users.md:352]: `let name = person.name` would also work

File: src/cheatsheets/gleam-for-erlang-users.md (hunk 1 above)
Comment (devonestes) [reply src/cheatsheets/gleam-for-erlang-users.md:216]: That looks a whole lot like a function call and not a literal... I guess there‘s no a reason why that can’t be a literal, but to someone just coming in to the language that would surely be mistaken as a function call!

Comment (lpil) [reply src/cheatsheets/gleam-for-erlang-users.md]: It's a similar syntax, but it's not a function. You can't assign the `tuple` keyword to a variable, for example.

**Pull request merged**